pkg/query-service/app/http_handler.go

```diff
@@ -2364,7 +2364,7 @@ func (aH *APIHandler) changePassword(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := auth.ChangePassword(context.Background(), req); err != nil {
-		if aH.HandleError(w, err, http.StatusInternalServerError) {
+		if aH.HandleError(w, err, http.StatusForbidden) {
```

Comment:
There can be other cases as well.
Only return status forbidden only when password is invalid https://github.com/SigNoz/signoz/blob/0df86454ce42796ec9b1015aa0fadd6141f254a1/pkg/query-service/auth/auth.go#L245